ironfish/src/network/messages/pooledTransactionsRequest.test.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { blake3 } from '@napi-rs/blake-hash'
+import { v4 as uuid } from 'uuid'
+import { PooledTrasactionsRequest } from './pooledTransactionsRequest'
+
+describe('PooledTrasactionsRequestMessage', () => {
+  it('serializes the object into a buffer and deserializes to the original object', () => {
+    const rpcId = 53242
+    const hashes = [...Array(10)].map((_) => blake3(uuid()))
```

Comment:
I'm guessing this test runs in like 100ms or so? If it's slower than that, we could probably just skip the hashing and create a 32-byte buffer directly, otherwise I think it's fine.

Reply:
Yea I'll create a 32 byte buffer

Reply:
Did some testing on my computer, with a list of size 10k it only took 15ms to do all the uuid and hashes. Seems to be pretty fast